### PR TITLE
Adding Multus to the list of Network model

### PR DESCRIPTION
**Pull request by rkamudhan**
Adding Multus (a multi plugin) to the list of Network model

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/5967)

### CODE REVIEW

File: docs/concepts/cluster-administration/networking.md [hunk 1]
```diff
@@ -184,6 +184,12 @@ Follow the "With Linux Bridge devices" section of [this very nice
 tutorial](http://blog.oddbit.com/2014/08/11/four-ways-to-connect-a-docker/) from
 Lars Kellogg-Stedman.
 
+### Multus (a Multi Network plugin)
+
+[Multus](https://github.com/Intel-Corp/multus-cni) is a Multi CNI plugin to support the Multi Networking feature in Kubernetes using CRD based network objects in Kubernetes.
```
Comment (xiangpengzhao) [inline docs/concepts/cluster-administration/networking.md]: Multi --> Multiple ?
Multi is OK though :)

File: docs/concepts/cluster-administration/networking.md [hunk 2]
```diff
@@ -184,6 +184,12 @@ Follow the "With Linux Bridge devices" section of [this very nice
...
+
+Multus support all [reference plugins](https://github.com/containernetworking/plugins) (eg. [Flannel](https://github.com/containernetworking/plugins/tree/master/plugins/meta/flannel), [DHCP](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/dhcp), [Macvlan](https://github.com/containernetworking/plugins/tree/master/plugins/main/macvlan)) that implement the CNI specification and 3rd party plugins (eg. [Calico](https://github.com/projectcalico/cni-plugin), [Weave](https://github.com/weaveworks/weave), [Cilium](https://github.com/cilium/cilium), [Contiv](https://github.com/contiv/netplugin)). In addition to it, Multus support [SRIOV](https://github.com/hustcat/sriov-cni), [DPDK](https://github.com/Intel-Corp/sriov-cni), [OVS-DPDK & VPP](https://github.com/intel/vhost-user-net-plugin) workloads in Kubernetes with both cloud native and NFV based application in Kubernetes.
```
Comment (xiangpengzhao) [inline docs/concepts/cluster-administration/networking.md]: support --> supports
application --> applications

File: docs/concepts/cluster-administration/networking.md (hunk 1 above)
Comment (rkamudhan) [reply docs/concepts/cluster-administration/networking.md]: yes yes, Multi is good :)

File: docs/concepts/cluster-administration/networking.md (hunk 2 above)
Comment (rkamudhan) [reply docs/concepts/cluster-administration/networking.md]: Will change it

**Comment by rkamudhan** [comment]
@xiangpengzhao Updated the changes 👍

**Comment by xiangpengzhao** [comment]
LGTM. Thanks for adding Multus here!

**Pull request merged**